app/assets/javascripts/discourse/tests/unit/lib/tracked-tools-test.gjs

```diff
@@ -0,0 +1,56 @@
+import { render, settled } from "@ember/test-helpers";
+import { module, test } from "qunit";
+import { dedupeTracked } from "discourse/lib/tracked-tools";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+
+module("Unit | tracked-tools", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("@dedupeTracked", async function (assert) {
+    class Pet {
+      initialsEvaluatedCount = 0;
+
+      @dedupeTracked name;
+
+      get initials() {
+        this.initialsEvaluatedCount++;
+        return this.name
+          ?.split(" ")
+          .map((n) => n[0])
+          .join("");
+      }
+    }
+
+    const pet = new Pet();
+    pet.name = "Scooby Doo";
+
+    await render(<template>
+      <span id="initials">{{pet.initials}}</span>
+    </template>);
+
+    assert.dom("#initials").hasText("SD", "Initials are correct");
```

Comment:
is rendering a template necessary to this test? 😃 

```suggestion
    assert.strictEqual(pet.initials, "SD", "Initials are correct");
```

Reply:
Hmm... we need some way to know whether downstream autotracking is being notified of a change. Rendering is one way to do that. But perhaps we get the same effect by using `@cached` on the getter. Lemme try 👀

Reply:
Ok that's much cleaner with `@cached`. Thanks @cvx!